spec/values/calculation/sqrt.hrx

```diff
@@ -0,0 +1,173 @@
+<===> README.md
+Most of the same behavior tested for `calc()` applies to `sqrt()`, but for
+terseness' sake isn't tested explicitly.
+
+<===>
+================================================================================
+<===> error/syntax/no_arg/input.scss
+a {b: calc(sqrt())}
```

Comment:
These do not need to be nested in another calc

Reply:
👍